### PR TITLE
Fix tthe Issue #45, please update it.

### PR DESCRIPTION
**Pull request by jacobyxu**
Hi Lchiffon,

I finally fix the firefox deploy issue and multiple plot click issue. please update it.

Thank you!

Best,

Puxin

### CODE REVIEW

**Comment by jacobyxu** [comment]
Test results:

feature          | firefox | chrome
---------------|------|---------
hover info  | work | work
multiple plot | work | work
separate click | work | work
special shape | work | not

**Comment by Lchiffon** [comment]
Thanks @JacobXPX could you please resolve these conflicts?


Conflicting files
DESCRIPTION
R/renderWordcloud2.R
man/wordcloud2-shiny.Rd
readme.md

**Comment by jacobyxu** [comment]
You mean change them back to what in the master branch now?

**Comment by Lchiffon** [comment]
image

@JacobXPX Your branch is behind the latest master branch, so there are some conflict in this repo,
you can resolve it in the [web editor](https://github.com/Lchiffon/wordcloud2/pull/46/conflicts)

for Example：

```
<<<<<<< master
wordcloud2Output(outputId, width = "100\%", height = "400px")
clickedWordInputId is automatically generated by: paste0(outputId, "_clicked_word"))

=======
wordcloud2Output(outputId, width = "100\%", height = "400px",
                 clickedWordInputId = "selected_word")
>>>>>>> master
```

should change to:

```
wordcloud2Output(outputId, width = "100\%", height = "400px",
                clickedWordInputId = "selected_word")
```



maybe next time you could run these code firstly:

```
git pull https://github.com/Lchiffon/wordcloud2
git push
```

**Comment by jacobyxu** [comment]
In your example, since I want to keep:

`wordcloud2Output(outputId, width = "100\%", height = "400px")
clickedWordInputId is automatically generated by: paste0(outputId, "_clicked_word"))`

I should just change it to this above, right?

Thank you for teaching me this. I think it should be fine now.

File: R/renderWordcloud2.R [hunk 1]
```diff
@@ -4,15 +4,15 @@
 ##' Function for plotting wordcloud2 in shiny
 ##'
 ##' @usage
-##' wordcloud2Output(outputId, width = "100\%", height = "400px",
-##'                  clickedWordInputId = "selected_word")
+##' wordcloud2Output(outputId, width = "100\%", height = "400px")
+##' clickedWordInputId is automatically generated by: paste0(outputId, "_clicked_word"))
+##'
```
Comment (Lchiffon) [inline R/renderWordcloud2.R]: If `clickedWordInputId` is not a param in the function, it shouldn't in @usage part.

Please add these to @details part

**Comment by Lchiffon** [comment]
And one more thing, if you add a file to the repo (like `examples/img/new.gif`) which will not used in R package, you should add the file name in `.Rbuildignore` 

https://github.com/Lchiffon/wordcloud2/blob/e92f89fd5d877abef5db22f6be35b98a8000c927/.Rbuildignore#L8

After finish `.Rbuildignore` , I think I can merge this PR

**Comment by Lchiffon** [comment]
Thanks~

**Comment by jacobyxu** [comment]
Thank you! learned a lot from you ;)

**Pull request merged**